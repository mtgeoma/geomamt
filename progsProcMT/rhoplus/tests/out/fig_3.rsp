  48.00       9.246     0.4241      9.045      48.50      1.300      47.55    
  36.00       8.757     0.3502      8.817      44.70      1.150      46.36    
  24.00       8.670     0.4046      8.889      45.80      1.300      45.20    
  18.00       8.770     0.4423      9.009      44.60      1.450      45.02    
  12.00       9.026     0.3307      9.055      46.70      1.050      45.30    
  9.000       9.018     0.4067      8.947      44.90      1.300      45.57    
  6.000       9.123     0.6123      8.565      45.70      1.900      45.48    
  4.500       8.011         0.      8.196      39.00         0.      44.54    
  3.000       7.837     0.1405      7.841      41.50     0.5000      41.44    
  2.250       7.918     0.1520      7.918      38.10     0.5500      38.23    
  1.500       8.677     0.2305      8.678      33.70     0.7500      33.51    
  1.125       9.754     0.2910      9.686      30.60     0.8500      30.73    
 0.7502       11.60     0.6900      11.70      27.80      1.700      27.91    
 0.5624       13.66     0.5750      13.60      26.60      1.200      26.56    
 0.3750       17.40      1.845      17.21      24.80      3.050      25.65    
 0.2813       21.36      1.915      20.59      25.70      2.550      26.05    
 0.1875       25.61      3.525      26.30      26.00      3.950      28.57    
 0.1406       30.65      2.300      30.34      30.50      2.150      31.71    
 9.3721E-02   33.15      3.345      34.45      34.60      2.900      37.25    
 7.0323E-02   33.94      2.540      35.65      43.00      2.150      41.20    
 4.6882E-02   36.65      4.855      35.46      45.30      3.750      46.10    
 3.5162E-02   35.04      4.260      34.60      48.70      3.500      49.18    
 2.3441E-02   32.04      4.320      32.62      54.00      3.850      53.47    
 1.7581E-02   30.02      4.150      30.59      56.60      3.950      56.52    
 1.1721E-02   26.97      3.850      27.14      61.30      4.100      60.79    
 8.7873E-03   24.92      3.695      24.48      63.90      4.200      63.97    
