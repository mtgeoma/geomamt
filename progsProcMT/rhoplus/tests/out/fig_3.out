Rho+ 2.0 29-4-97
 
==> data *           
 
Data read from file *                                                                          
    Freq (Hz) rho(ohm-m)  rhoerr    lg10rho lg10err phi(deg)  err   downweight
  1  48.0       6.878      0.316      0.837  0.020   48.50    1.30  1.00  1.00
  2  36.0       6.514      0.261      0.814  0.017   44.70    1.15  1.00  1.00
  3  24.0       6.449      0.301      0.809  0.020   45.80    1.30  1.00  1.00
  4  18.0       6.524      0.329      0.815  0.022   44.60    1.45  1.00  1.00
  5  12.0       6.714      0.246      0.827  0.016   46.70    1.05  1.00  1.00
  6  9.00       6.708      0.302      0.827  0.020   44.90    1.30  1.00  1.00
  7  6.00       6.786      0.456      0.832  0.029   45.70    1.90  1.00  1.00
  8  4.50       5.959      0.389      0.775  0.028   39.00    1.90  0.00  0.00
  9  3.00       7.837      0.141      0.894  0.008   41.50    0.50  1.00  1.00
 10  2.25       7.918      0.152      0.899  0.008   38.10    0.55  1.00  1.00
 11  1.50       8.677      0.231      0.938  0.012   33.70    0.75  1.00  1.00
 12  1.12       9.754      0.291      0.989  0.013   30.60    0.85  1.00  1.00
 13 0.750       11.60      0.690      1.064  0.026   27.80    1.70  1.00  1.00
 14 0.562       13.66      0.575      1.135  0.018   26.60    1.20  1.00  1.00
 15 0.375       17.40       1.84      1.241  0.046   24.80    3.05  1.00  1.00
 16 0.281       21.36       1.92      1.330  0.039   25.70    2.55  1.00  1.00
 17 0.188       25.61       3.52      1.408  0.060   26.00    3.95  1.00  1.00
 18 0.141       30.65       2.30      1.486  0.033   30.50    2.15  1.00  1.00
 19 9.372E-02   33.15       3.35      1.520  0.044   34.60    2.90  1.00  1.00
 20 7.032E-02   33.94       2.54      1.531  0.033   43.00    2.15  1.00  1.00
 21 4.688E-02   36.65       4.86      1.564  0.058   45.30    3.75  1.00  1.00
 22 3.516E-02   35.04       4.26      1.545  0.053   48.70    3.50  1.00  1.00
 23 2.344E-02   32.04       4.32      1.506  0.059   54.00    3.85  1.00  1.00
 24 1.758E-02   30.02       4.15      1.477  0.060   56.60    3.95  1.00  1.00
 25 1.172E-02   26.97       3.85      1.431  0.062   61.30    4.10  1.00  1.00
 26 8.787E-03   24.92       3.69      1.397  0.064   63.90    4.20  1.00  1.00
 
==> bands 9 26 1 8 -1 
Phase data not pruned outside bands
 
==> surface conduct  

Rho and phase used in band gain calculation
Gains judged using fit to rho & phase

Calculated optimum gains multiplied times rho data:
 Band   9 : 26  1.0000 Fixed
 Band   1 :  8  1.3443

Report: Before applying optimum gains: Chisq =   76.90    
 
            Table of misfits between data and model predictions
 
                ----- Apparent Resistivity ------   ---------- Phase ----------
    freq        data    predicted     %dR/R  norm   data  predicted   dP   norm
                                                                   (scaled)    
  1  48.0      6.878      6.830        -0.7   0.2   48.50   49.19     2.4   0.5
  2  36.0      6.514      6.862         5.3   1.3   44.70   47.01     8.1   2.0
  3  24.0      6.449      7.038         9.1   1.9   45.80   45.19    -2.1   0.5
  4  18.0      6.524      7.124         9.2   1.7   44.60   44.38    -0.8   0.2
  5  12.0      6.714      7.218         7.5   2.0   46.70   43.15   -12.4   3.4
  6  9.00      6.708      7.352         9.6   2.0   44.90   42.21    -9.4   2.1
  7  6.00      6.786      7.626        12.4   1.7   45.70   41.24   -15.6   2.3
  8  4.50         --      7.766          -     -     --     40.77     -      - 
  9  3.00      7.837      7.803        -0.4   0.2   41.50   39.49    -7.0   4.0
 10  2.25      7.918      7.853        -0.8   0.4   38.10   37.58    -1.8   0.9
 11  1.50      8.677      8.333        -4.0   1.5   33.70   33.62    -0.3   0.1
 12  1.12      9.754      9.154        -6.2   2.1   30.60   30.63     0.1   0.0
 13 0.750      11.60      11.11        -4.2   0.7   27.80   27.38    -1.5   0.2
 14 0.562      13.66      13.06        -4.4   1.1   26.60   26.07    -1.9   0.4
 15 0.375      17.40      16.58        -4.7   0.5   24.80   25.54     2.6   0.2
 16 0.281      21.36      19.69        -7.8   0.9   25.70   26.05     1.2   0.1
 17 0.188      25.61      24.96        -2.5   0.2   26.00   28.31     8.1   0.6
 18 0.141      30.65      28.94        -5.6   0.8   30.50   31.17     2.3   0.3
 19 9.372E-02  33.15      33.42         0.8   0.1   34.60   36.61     7.0   0.7
 20 7.032E-02  33.94      34.93         2.9   0.4   43.00   40.81    -7.6   1.0
 21 4.688E-02  36.65      34.70        -5.3   0.4   45.30   46.14     2.9   0.2
 22 3.516E-02  35.04      33.61        -4.1   0.3   48.70   49.30     2.1   0.2
 23 2.344E-02  32.04      31.62        -1.3   0.1   54.00   53.36    -2.2   0.2
 24 1.758E-02  30.02      29.91        -0.4   0.0   56.60   56.35    -0.9   0.1
 25 1.172E-02  26.97      26.89        -0.3   0.0   61.30   61.02    -1.0   0.1
 26 8.787E-03  24.92      24.23        -2.8   0.2   63.90   64.73     2.9   0.2

 - norm: missfit divided by data error (as modified by downweights)
 - %dR/R: (rho missfit)/rho in %
 - dP(scaled): (phase missfit) * 3.491 to give it same scale as %dR/R


Report: Before applying optimum gains: Chisq =   76.90    

        After applying optimum gains: Chisq =   10.55    
 
            Table of misfits between data and model predictions
 
                ----- Apparent Resistivity ------   ---------- Phase ----------
    freq        data    predicted     %dR/R  norm   data  predicted   dP   norm
                                                                   (scaled)    
  1  48.0      9.246      9.045        -2.2   0.5   48.50   47.55    -3.3   0.7
  2  36.0      8.757      8.817         0.7   0.2   44.70   46.36     5.8   1.4
  3  24.0      8.670      8.889         2.5   0.5   45.80   45.20    -2.1   0.5
  4  18.0      8.770      9.009         2.7   0.5   44.60   45.02     1.5   0.3
  5  12.0      9.026      9.055         0.3   0.1   46.70   45.30    -4.9   1.3
  6  9.00      9.018      8.947        -0.8   0.2   44.90   45.57     2.3   0.5
  7  6.00      9.123      8.565        -6.1   0.9   45.70   45.48    -0.8   0.1
  8  4.50         --      8.196          -     -     --     44.54     -      - 
  9  3.00      7.837      7.841         0.1   0.0   41.50   41.44    -0.2   0.1
 10  2.25      7.918      7.918         0.0   0.0   38.10   38.23     0.4   0.2
 11  1.50      8.677      8.678         0.0   0.0   33.70   33.51    -0.7   0.2
 12  1.12      9.754      9.686        -0.7   0.2   30.60   30.73     0.4   0.2
 13 0.750      11.60      11.70         0.9   0.2   27.80   27.91     0.4   0.1
 14 0.562      13.66      13.60        -0.4   0.1   26.60   26.56    -0.2   0.0
 15 0.375      17.40      17.21        -1.1   0.1   24.80   25.65     3.0   0.3
 16 0.281      21.36      20.59        -3.6   0.4   25.70   26.05     1.2   0.1
 17 0.188      25.61      26.30         2.7   0.2   26.00   28.57     9.0   0.7
 18 0.141      30.65      30.34        -1.0   0.1   30.50   31.71     4.2   0.6
 19 9.372E-02  33.15      34.45         3.9   0.4   34.60   37.25     9.2   0.9
 20 7.032E-02  33.94      35.65         5.0   0.7   43.00   41.20    -6.3   0.8
 21 4.688E-02  36.65      35.46        -3.2   0.2   45.30   46.10     2.8   0.2
 22 3.516E-02  35.04      34.60        -1.2   0.1   48.70   49.18     1.7   0.1
 23 2.344E-02  32.04      32.62         1.8   0.1   54.00   53.47    -1.9   0.1
 24 1.758E-02  30.02      30.59         1.9   0.1   56.60   56.52    -0.3   0.0
 25 1.172E-02  26.97      27.14         0.6   0.0   61.30   60.79    -1.8   0.1
 26 8.787E-03  24.92      24.48        -1.8   0.1   63.90   63.97     0.3   0.0

 - norm: missfit divided by data error (as modified by downweights)
 - %dR/R: (rho missfit)/rho in %
 - dP(scaled): (phase missfit) * 3.491 to give it same scale as %dR/R

 
==> matrix           

Response summary matrix written to fig_3.rsp
 
==> model            

Final Refinement: Chisq =   10.55    

                              Poles
 4.21549E-02 0.15198     0.60047      4.7608      42.695      124.00    
  1338.3    

                              Zeros
      0.     6.58612E-02 0.27279      3.4532      15.016      73.643    
  290.41    


   a0 =          0.    
    n    lambda(n)          a(n)
    1     4.21549E-02      471.57    
    2     0.15198          1100.6    
    3     0.60047          3407.6    
    4      4.7608          1647.8    
    5      42.695          7980.0    
    6      124.00          12245.    
    7      1338.3         1.34899E+05

     Units are kilometers and siemens
level  depth      conductance   separation   total conductance
  1        0.        4.9197            0.        4.9197    
  2   0.14344        28.018       0.14344        32.938    
  3   0.50984        51.861       0.36640        84.798    
  4    1.0715        47.405       0.56168        132.20    
  5    5.4417        74.031        4.3702        206.24    
  6    12.462        631.58        7.0207        837.82    
  7    20.697        3183.6        8.2341        4021.5    
       24.836          --         4.1395    
 Model terminates with a perfect conductor
 
==> task *           
 
==> criterion 0.95   

Just acceptable chi-squared level:   67.505
Confidence level of this misfit:      0.950
 
==> matrix           
 
==> root fig_3
             
                      Bounds on Phase (degrees)
             
              frequency   period       lower       upper
               48.00      2.0833E-02   39.91       54.72      
               36.00      2.7778E-02   41.07       51.75      
               24.00      4.1667E-02   41.08       49.61      
               18.00      5.5556E-02   41.02       49.15      
               12.00      8.3333E-02   41.40       49.30      
               9.000      0.1111       41.86       49.46      
               6.000      0.1667       42.07       48.86      
               4.500      0.2222       41.32       47.35      
               3.000      0.3333       38.76       43.78      
               2.250      0.4444       35.90       40.56      
               1.500      0.6667       31.11       36.26      
               1.125      0.8889       27.79       33.87      
              0.7502       1.333       23.99       31.60      
              0.5624       1.778       22.27       30.85      
              0.3750       2.667       20.76       31.05      
              0.2813       3.555       21.72       32.14      
              0.1875       5.333       23.51       35.11      
              0.1406       7.112       25.63       38.31      
              9.3721E-02   10.67       29.60       43.84      
              7.0323E-02   14.22       32.76       48.01      
              4.6882E-02   21.33       37.11       53.95      
              3.5162E-02   28.44       39.79       58.08      
              2.3441E-02   42.66       42.69       63.92      
              1.7581E-02   56.88       43.89       68.64      
              1.1721E-02   85.32       43.71       75.16      
              8.7873E-03   113.8       40.50       78.71      
             
                Bounds on Apparent Resistivity (ohm-m)
             
              frequency   period       lower       upper
               48.00      2.0833E-02   7.372       11.48      
               36.00      2.7778E-02   7.418       10.55      
               24.00      4.1667E-02   7.564       10.39      
               18.00      5.5556E-02   7.644       10.54      
               12.00      8.3333E-02   7.730       10.60      
               9.000      0.1111       7.666       10.33      
               6.000      0.1667       7.435       9.642      
               4.500      0.2222       7.256       9.116      
               3.000      0.3333       7.147       8.622      
               2.250      0.4444       7.286       8.654      
               1.500      0.6667       7.879       9.524      
               1.125      0.8889       8.614       10.72      
              0.7502       1.333       10.17       13.36      
              0.5624       1.778       11.68       16.02      
              0.3750       2.667       14.35       21.04      
              0.2813       3.555       16.61       25.46      
              0.1875       5.333       20.16       32.63      
              0.1406       7.112       22.71       37.91      
              9.3721E-02   10.67       25.49       44.52      
              7.0323E-02   14.22       26.29       47.82      
              4.6882E-02   21.33       25.82       49.62      
              3.5162E-02   28.44       24.68       49.41      
              2.3441E-02   42.66       22.25       47.63      
              1.7581E-02   56.88       20.09       45.95      
              1.1721E-02   85.32       16.08       42.87      
              8.7873E-03   113.8       13.09       40.63      

Bounds written in matrix form to fig_3.bnd
 
==> plot xyplot      
 
Plotfile type is: xyplot
 
Plotfile written to fig_3.plt
 
All requested rho+ computations finished
